src/text_object.h

```diff
@@ -76,25 +77,37 @@ void gen_print_nothing(struct text_object *, char *, int);
 void gen_print_obj_data_s(struct text_object *, char *, int);
 
 class legacy_cb: public conky::callback<void *, int (*)()> {
-    typedef conky::callback<void *, int (*)()> Base;
+	typedef conky::callback<void *, int (*)()> Base;
 
 protected:
-    virtual void work()
-    { std::get<0>(tuple)(); }
+	virtual void work()
+	{ std::get<0>(tuple)(); }
 
 public:
-    legacy_cb(uint32_t period, int (*fn)())
-        : Base(period, true, Base::Tuple(fn))
-    {}
+	legacy_cb(uint32_t period, int (*fn)())
+		: Base(period, true, Base::Tuple(fn))
+	{}
 };
+
 typedef conky::callback_handle<legacy_cb> legacy_cb_handle;
+typedef conky::callback_handle<exec_cb> exec_cb_handle;
 
+/**
+ * This is where Conky collects information on the conky.text objects in your config
+ *
+ * During startup and reload, objects are parsed and callbacks are set. Note that
+ * there are currently two types of callback: obj_cb (old style) and
+ * conky::callback (new style). On each update interval, generate_text_internal()
+ * in conky.cc traverses the list of text_objects and calls the old callbacks.
+ * The new style callbacks are run separately by conky::run_all_callbacks().
+ */
 struct text_object {
 	struct text_object *next, *prev;	/* doubly linked list of text objects */
-	struct text_object *sub;		/* for objects parsing text into objects */
+	struct text_object *sub;			/* for objects parsing text into objects */
```

Comment:
Yeah, I noticed that the tabs don't align in the diffs. The reason I changed these lines is that they were misaligned in my editors (Kate and vim), which both use a tab width of 4 characters. Should I set the tab width to 8 instead? In any case, I'll fix the spacing so they line up here. Thanks!